Latex/vorlage/vorlage.tex

```diff
@@ -190,16 +202,16 @@
 
 
 %! PageBreaks nach jeder Section
-\let\oldsection\section
-\renewcommand\section{\clearpage\oldsection}
+\let\oldchapter\chapter
+\renewcommand\chapter{\clearpage\oldchapter}
```

Comment:
iirc meinte einer von euch auf whatsapp, das hier gänge einfacher - hattet ihr das geklärt?

Reply:
haben wir, gehts nicht (afaik)

Reply:
Ja, die Vorlage verwendet irgendwie `\chapters` in section-style, weswegen es keinen Umbruch mehr durchführt...
Also muss der Befehl drin stehen bleiben.